### ISSUE TITLE
Require cycles (again)

### ISSUE THREAD

**Issue by felansu** [Dean177/react-native-json-tree#154]
Ref: https://github.com/Dean177/react-native-json-tree/issues/147

I get that warning message:

```
[Wed Sep 02 2020 07:14:43.397]  WARN     Require cycle: node_modules/react-native-json-tree/src/JSONNode.js -> node_modules/react-native-json-tree/src/JSONObjectNode.js -> node_modules/react-native-json-tree/src/JSONNestedNode.js -> node_modules/react-native-json-tree/src/JSONNode.js

Require cycles are allowed, but can result in uninitialized values. Consider refactoring to remove the need for a cycle.
```

**Comment by tannermares** [comment]
This warning is currently blocking me from updating to RN 0.64. I'm working on a PR right now.

**Comment by tannermares** [comment]
@Dean177 I tried taking a look at this yesterday for a couple hours. I don't know if it is a problem with my local machine, but I couldn't run the tests locally, and updating all of the deps to get the lib in a good state was a little more of a job than I had time to do right now. 

This warning is currently causing my RN 0.64 build to fail. If I comment out the only place I am importing `react-native-json-tree` everything loads per usual. I can try to duplicate with a fresh RN app but I think the fix should be pretty straightforward and useful regardless of RN version.

**Comment by Dean177** [comment]
@tannermares thanks for taking a look, did you manage to fix the warning even if the tests wouldn't run?

**Comment by tannermares** [comment]
@Dean177 After upgrading to RN latest I got everything compiling but was hitting a different error. [`TypeError expected a function`](https://github.com/Dean177/react-native-json-tree/issues/151) So I gave up 😂 . My first approach was going to put all of the `JSON` prefixed files into a `JSON` folder and adding an `index.js` that imports and exports all of those files. Then using that `index.js` as the method of access. If that didn't work then I would just put them all in a single file  😅

**Comment by tannermares** [comment]
Also, this library is really slick and does exactly what I wanted and more. So thanks for putting it together! 🤗

**Comment by felansu** [comment]
Thanks for not forgetting us

**Comment by Dean177** [comment]
This should be resolved in [1.2.1](https://www.npmjs.com/package/react-native-json-tree/v/1.2.1)
